lib/livebook/teams/deployment_group.ex

```diff
@@ -1,19 +1,22 @@
 defmodule Livebook.Teams.DeploymentGroup do
   use Ecto.Schema
   import Ecto.Changeset
+  alias Livebook.Secrets.Secret
 
   @type t :: %__MODULE__{
           id: pos_integer() | nil,
```

Comment:
```suggestion
          id: String.t() | nil,
```

and we need to update the schema.